src/template/layout/master.jade

```diff
@@ -34,8 +34,8 @@ html
                     ul.footer-list
                         li: a(href='#{beamUrl}/legal/tos') Terms of Service
                         li: a(href='#{beamUrl}/legal/privacy') Privacy Policy
-                        li: a(href='#{beamUrl}/contact') Contact
-                        li: a(href='#{beamUrl}') Beam
+                        li: a(href='https://beam.pro/contact/contact') Contact
```

Comment:
@ttaylorr it was undefined ![https://imnjb.me/i/fPF7Kg.png](https://imnjb.me/i/fPF7Kg.png)